build/webpack/webpack.common.config.js

```diff
@@ -5,9 +5,36 @@ const config = {
 	externals: require('./externals'),
 	appShell: require('./app-shell-entry-points'),
 	es5: require('./loaders/es5'),
+	es6: require('./loaders/es6'),
 	templates: require('./loaders/templates'),
 	text: require('./loaders/text')
 };
 
-module.exports = (configKeys = []) =>
+const webpackConfig = (configKeys = []) =>
 	webpackMerge(...configKeys.map(key => config[key]));
+
+function webpackConfigFormula ({
+	includeExternals = false,
+	includeAppShell = false,
+	jsLoader = 'es5'
```

Comment:
I ❤️ this! Using config objects with defaults!

Reply:
me too